rai/core_test/uint256_union.cpp

```diff
@@ -13,6 +13,33 @@ TEST (uint128_union, decode_dec)
     ASSERT_EQ (16, value.bytes [15]);
 }
 
+struct test_punct : std::moneypunct<char> {
+    pattern do_pos_format () const { return { {value, none, none, none} }; }
+    int do_frac_digits () const { return 0; }
+    char_type do_decimal_point () const { return '+'; }
+    char_type do_thousands_sep () const { return '-'; }
+    string_type do_grouping () const { return "\3\4"; }
+};
+
+TEST (uint128_union, balance_format)
```

Comment:
Can you add tests for `uint128_t` with all of the bits set, all of them except for the LSB, all of them except the MSB?

Reply:
Done, the 0xFF... case actually found an overflow - thanks :)